packages/panzer/disc-fe/src/evaluators/Panzer_ScatterResidual_Epetra_impl.hpp

```diff
@@ -418,11 +418,15 @@ evaluateFields(typename TRAITS::EvalData workset)
       std::size_t cellLocalId = localCellIds[worksetCellIndex];
 
       rLIDs = globalIndexer_->getElementLIDs(cellLocalId); 
-      cLIDs = colGlobalIndexer->getElementLIDs(cellLocalId);
+      initial_cLIDs = colGlobalIndexer->getElementLIDs(cellLocalId);
+      std::vector<int> cLIDs;
```

Comment:
What's the reason for making this `std::vector` of LIDs out of the `Kokkos::View` version?

Reply:
Because you need to add to it later in the code.  The other option is to count, allocate fill but the vector seems easier.